tests/ci/style_check.py

```diff
@@ -28,6 +28,13 @@
 
 NAME = "Style Check"
 
+GIT_PREFIX = (  # All commits to remote are done as robot-clickhouse
```

Comment:
It's the same in the whole https://github.com/ClickHouse/ClickHouse/blob/5d7cb6b85a75eb2e4db9305e113c8ed88a5ba753/tests/ci/cherry_pick.py, I'd like to keep it the same semantically. With `GIT_COMMAND_PREFIX` it would be too long to use